ConcernsCaseWork/ConcernsCaseWork/Helpers/Utilities.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.Helpers
+{
+	public class Utilities
```

Comment:
I would replace this with an extension method, which follows other ways of working. Use the StringExtension :) thoughts